src/deps.ts

```diff
@@ -6,4 +6,5 @@ export * as path from "https://deno.land/std@0.170.0/path/mod.ts";
 export { readAll } from "https://deno.land/std@0.170.0/streams/read_all.ts";
 export { writeAllSync } from "https://deno.land/std@0.170.0/streams/write_all.ts";
 export { default as localDataDir } from "https://deno.land/x/dir@1.5.1/data_local_dir/mod.ts";
+export { outdent } from "https://deno.land/x/outdent@v0.8.0/mod.ts";
```

Comment:
```suggestion
export { outdent } from "https://deno.land/x/outdent@v0.8.0/src/index.ts";
```

The mod.ts file seems to just be a re-export of the src/index.ts file, so we can bypass it.